docs/readthedocs/source/doc/LLM/Quickstart/fastchat_quickstart.md

```diff
@@ -0,0 +1,96 @@
+# Serving using IPEX-LLM and FastChat
+
+FastChat is an open platform for training, serving, and evaluating large language model based chatbots. You can find the detailed information at their [homepage](https://github.com/lm-sys/FastChat).
+
+IPEX-LLM can be easily integrated into FastChat so that user can use `IPEX-LLM` as a serving backend in the deployment.
+
+## Quick Start
+This quickstart guide walks you through installing and running `FastChat` with `ipex-llm`.
+
+## 1. Install IPEX-LLM with FastChat
+
+```bash
+pip install --pre --upgrade ipex-llm[serving]
```

Comment:
Using `pip install --pre --upgrade ipex-llm[serving]` to install will not install all the required packages for running `ipex-llm`.

I guess install using `pip install --pre --upgrade ipex-llm[all,serving]`?